_posts/2016-02-11-GitHub.md

```diff
@@ -0,0 +1,23 @@
+---  
+layout: default  
+title: 'Brief overview of GitHub'  
+categories: [blog]  
+date: 2016-02-16 00:00:00 -0800  
+author: kathweaver  
+download: true  
+---
+
+For those new to GitHub, and I am one, it's very well documented.  First a quick overview of the process.
+
+Open Live Writer has two different repositories (a repository is just a place to store things).  One for the program itself
+and a second one for the documentation.  If you are reading this, you are reading documentation.
```

Comment:
Is there a reason you're forcibly splitting sentences? This should all just be one line, right?